inlong-sort/sort-connectors/doris/src/main/java/org/apache/inlong/sort/doris/table/DorisDynamicSchemaOutputFormat.java

```diff
@@ -493,8 +690,12 @@ public static class Builder {
         private String databasePattern;
         private String tablePattern;
         private boolean ignoreSingleTableErrors;
+        private boolean isSingle;
```

Comment:
a boolean field start with is not recomment

Reply:
fixed